thunder/executors/nvfuserex_impl.py

```diff
@@ -2847,6 +2847,11 @@ def scatter(
     nvi = getnv(index, fd, lc_to_nv_map)
     nvs = getnv(src, fd, lc_to_nv_map)
 
+    # NOTE: scalar source value is not supported until 0.2.31. Wrapping scalar to full as a WAR.
```

Comment:
I hope you don't mind me directly pushing here.

Reply:
I don't mind at all! Rather this is helpful, thank you!

Reply:
Oops, sorry let me take a look at that...

Reply:
missed the message from Jingyue above.

That's already patched on main, so we just needed to bump our stable wheel.

Reply:
hold on. The failure doesn't make sense... Even with the old commit without the version bump, the fix should be in already...

Let me pull a container to try it.

Reply:
nvm. pypi is updated. but looks like thunder CI is still pulling the pypi wheel somehow. I should have checked the log.

Reply:
for some major packages like Torch we need Docker build for some others it shall be updated with CI, and (and Docker build is just making the CI faster)